base-theme/layouts/partials/seo.html

```diff
@@ -7,11 +13,17 @@
   {{- else -}}
   {{- $metaDataDescription = (.Params.description | plainify) | default $defaultMetadataDescription -}}
   {{- end -}}
-  {{- $sitemapDomain := partial "sitemap_domain.html" -}}
-  {{- $path := partial "site_root_url.html" .RelPermalink -}}
 
   <meta name="description" content="{{- $metaDataDescription -}}">
   <meta name="keywords" content="opencourseware,MIT OCW,courseware,MIT opencourseware,Free Courses,class notes,class syllabus,class materials,tutorials,online courses,MIT courses">
+  {{/*  Facebook Open Graph  */}}
+  <meta property="og:site_name" content="MIT OpenCourseWare">
+  <meta property="og:type" content="website" />
+  <meta property="og:url" content="https://{{ strings.TrimSuffix "/" $sitemapDomain }}/{{- strings.TrimPrefix "/" $path -}}" />
+  <meta property="og:title" content="{{- .Site.Data.course.course_title -}}" />
+  <meta property="og:description" content="{{- $metaDataDescription -}}" />
+  <meta property="og:image" content="{{- $courseImageUrl -}}" />
+  
```

Comment:
@pdpinch 
This works fine for courses However since these changes were moved to the base theme, ocw-www also renders meta tags and those do not posses valid values. Is metadata a concern for ocw-www and if so which pages should we concern ourselves elfs with ?

<img width="963" alt="Screenshot 2022-08-19 at 7 36 29 PM" src="https://user-images.githubusercontent.com/87968618/185642836-482fe2d5-c280-4b90-bedd-1307b93abac6.png">

Reply:
Ideally ocw-www would get it's own set of metadata. Do we have things set up now so that ocw-www can override the base theme?

Reply:
@pdpinch We can do that.
Can you provide `ocw-www` metadata?

Reply:
```
  <meta property="og:site_name" content="MIT OpenCourseWare">
  <meta property="og:type" content="website" />
  <meta property="og:url" content="https://{{ strings.TrimSuffix "/" $sitemapDomain }}/" />
  <meta property="og:title" content="[Page title]" />
  <meta property="og:description" content="{{- $metaDataDescription -}}" />
  <meta property="og:image" content="{{- $courseImageUrl -}}" />
```

- I'm assuming that you can pull the page title and description from the front matter. If you need some for the static pages let me know
- Is there an easy way to pull the first image on the page for the image? If not, use a (larger) OCW logo